<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>viewMaterial</class>
 <widget class="QWidget" name="viewMaterial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>530</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>940</width>
    <height>530</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>material details</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget { background-color: white; font-family: &quot;Poppins&quot;, Arial, sans-serif; }</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="headerLayout">
       <item>
        <widget class="QLabel" name="backButton">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border: none; background: transparent; padding: 5px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../../../../../../src/icons.qrc">:/icons/back2.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="titleMetaLayout">
         <item>
          <widget class="QLabel" name="title_label">
           <property name="styleSheet">
            <string notr="true">font-size: 54px; font-weight: 400; color: #333; margin-bottom: 10px;</string>
           </property>
           <property name="text">
            <string>Desktop Project Guidelines</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="metaLayout">
           <item>
            <widget class="QLabel" name="instructor_label">
             <property name="styleSheet">
              <string notr="true">font-size: 20px; color: #24292f;
margin-left: 10px;</string>
             </property>
             <property name="text">
              <string>Carlos Fidel Castro</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="date_label">
             <property name="styleSheet">
              <string notr="true">font-size: 20px; color: #656d76; padding-left: 5px;</string>
             </property>
             <property name="text">
              <string>• August 18, 2025</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="menuButton">
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background: transparent;
    border: none;
    color: #6c757d;
    font-size: 40px;
    font-weight: bold;
    border-radius: 20px;
}
QPushButton:hover {
    background-color: #f8f9fa;
    color: #495057;
}
QPushButton:pressed {
    background-color: #e9ecef;
}</string>
         </property>
         <property name="text">
          <string>⋮</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="descriptionEdit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>900</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 16px; color: #000000; padding: 10px;  border: 1px solid #ddd; border-radius: 5px;
margin-left: 80px;
margin-bottom: 10px;</string>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::WidgetWidth</enum>
     </property>
     <property name="lineWrapColumnOrWidth">
      <number>0</number>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Poppins,Arial,sans-serif'; font-size:16px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14px;&quot;&gt;Please ensure that the task is completed according to the requirements provided, keeping everything consistent and aligned throughout the process. Make sure to follow the necessary steps as outlined, review your work before finalizing, and confirm that it meets the expected standards. Always double-check that the format is correct, the details are accurate, and the output follows the general guidelines. Be mindful of maintaining a clear structure, avoid unnecessary errors, and ensure that the submission is properly prepared before turning it in.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="attachementLayout" stretch="0,0">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>80</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QFrame" name="attachmentFrame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>76</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>76</height>
        </size>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame #attachmentFrame { 
background-color: white; 
border: 1px solid #ddd; 
border-radius: 10px; 
padding: 5px;}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="attachmentInnerLayout">
          <item>
           <widget class="QLabel" name="attachmentName">
            <property name="styleSheet">
             <string notr="true">
font-size: 16px; 
color: #24292f; 
text-align: center; 
text-decoration: underline;
</string>
            </property>
            <property name="text">
             <string>Desktop project guidelines</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="attachmentType">
            <property name="styleSheet">
             <string notr="true">font-size: 14px; color: #656d76; text-align: center;</string>
            </property>
            <property name="text">
             <string>PDF</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="commentLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>38</width>
         <height>38</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #084924; border-radius: 19px; color: white; min-width: 38px; min-height: 38px; text-align: center; line-height: 38px;</string>
       </property>
       <property name="text">
        <string>C</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="commentBox">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>38</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 14px; color: #24292f; padding: 5px; border: 1px solid #ddd; border-radius: 16.5px;</string>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Poppins,Arial,sans-serif'; font-size:14px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Add a comment...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>60</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../../../src/font.qrc"/>
  <include location="../../../../../../src/icons.qrc"/>
 </resources>
 <connections/>
</ui>
